src/DssSpell.sol

```diff
@@ -65,99 +47,89 @@ contract DssSpellAction is DssAction {
-    uint256 constant internal STARKNET_L2_SPELL = 0x4e7d83cd693f8b518f9638ce47d573fd2d642371ee266d6ed55e1276d5b43c3;
-
-    // run ./scripts/get-opt-relay-cost.sh to help determine Optimism relay param
-    uint32 public constant OPT_MAX_GAS = 100_000; // = 44582 gas (estimated L2 execution cost) + margin
-
-    // run ./scripts/get-arb-relay-cost.sh to help determine Arbitrum relay params
-    uint256 public constant ARB_MAX_GAS = 100_000; // = 38_920 gas (estimated L1 calldata + L2 execution cost) + margin (to account for surge in L1 basefee)
-    uint256 public constant ARB_GAS_PRICE_BID = 1_000_000_000; // = 0.1 gwei + 0.9 gwei margin
-    uint256 public constant ARB_MAX_SUBMISSION_COST = 1e15; // = ~0.7 * 10^15 (@ ~15 gwei L1 basefee) rounded up to 1*10^15
-    uint256 public constant ARB_L1_CALL_VALUE = ARB_MAX_SUBMISSION_COST + ARB_MAX_GAS * ARB_GAS_PRICE_BID;
-
-    // see: https://github.com/makerdao/starknet-spells-mainnet/blob/55401e8121f93d09f57f61c4e77dc0b6c73fb4f8/README.md#estimate-l1-l2-fee
-    uint256 public constant STA_GAS_USAGE_ESTIMATION = 28460;
-
-    // 500gwei, ~upper bound of monthly avg gas price in `21-`22,
-    // ~100x max monthly median gas price in `21-`22
-    // https://explorer.bitquery.io/ethereum/gas?from=2021-01-01&till=2023-01-31
-    uint256 public constant STA_GAS_PRICE = 500000000000;
-    uint256 public constant STA_L1_CALL_VALUE = STA_GAS_USAGE_ESTIMATION * STA_GAS_PRICE;
 
     function actions() public override {
-        // ------------------ Pause Optimism Goerli L2DaiTeleportGateway -----------------
-        // Forum: https://forum.makerdao.com/t/community-notice-pecu-to-redeploy-teleport-l2-gateways/19550
-        // L2 Spell to execute via OPTIMISM_GOV_RELAY:
-        // https://optimistic.etherscan.io/address/0x9495632f53cc16324d2fcfcdd4eb59fb88ddab12#code
-        OptimismGovRelayLike(OPTIMISM_GOV_RELAY).relay(
-            OPTIMISM_L2_SPELL,
-            abi.encodeWithSignature("execute()"),
-            OPT_MAX_GAS
+        // Dust Parameter Changes
+        // https://vote.makerdao.com/polling/QmRfegL4#vote-breakdown
+
+        // Reduce dust for ETH-A, WBTC-A, and WSTETH-A to 7,500 DAI.
+        DssExecLib.setIlkMinVaultAmount("ETH-A", 7_500);
+        DssExecLib.setIlkMinVaultAmount("WBTC-A", 7_500);
+        DssExecLib.setIlkMinVaultAmount("WSTETH-A", 7_500);
+
+        // Reduce dust for ETH-C, WBTC-C, and WSTETH-B to 3,500 DAI.
+        DssExecLib.setIlkMinVaultAmount("ETH-C", 3_500);
+        DssExecLib.setIlkMinVaultAmount("WBTC-C", 3_500);
+        DssExecLib.setIlkMinVaultAmount("WSTETH-B", 3_500);
+
+        // Reduce dust for ETH-B and WBTC-B to 25,000 DAI.
+        DssExecLib.setIlkMinVaultAmount("ETH-B", 25_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-B", 25_000);
+
+
+        // Chainlink Automation Keeper Network Stream Setup
+        // https://vote.makerdao.com/polling/QmXeWcrX
+        // Note: unrestricted stream
+        VestLike(MCD_VEST_DAI).create(
```

Comment:
Should verify but this probably needs to be restricted so only the receiver contract can vest according to its logic.

Reply:
Yeah I left this unrestricted because it's a keeper so that anyone could top it up, but if it's maintaining it's own logic we should restrict.

Reply:
Also wondering now if we should make the PP the manager so that the DAO retains the authority to yank. Will ask.

Reply:
Isn't the PP still a ward? (if so it can still yank without being the manager)

Reply:
Added restricted logic. I'd forgotten that I wrote the code to allow the PP to yank regardless. :sweat_smile: